crates/frontend/src/components/table.rs

```diff
@@ -80,9 +65,8 @@ pub fn table(
                                         .filter(|column| !column.hidden)
                                         .map(|column| {
                                             let cname = &column.name;
-                                            let value: String = generate_table_row_str(
-                                                row.get(cname).unwrap_or(&Value::String("".to_string())),
-                                            );
+                                            let value: String =
+                                                row.get(cname).unwrap_or(&Value::String(String::new())).html_display();
```

Comment:
what difference is `html_display` creating here ?

Reply:
It's similar to earlier implementation. Just using `html_display` as a standard across the frontend.

Reply:
This is similar to `generate_table_row_str`, just using html_display everywhere for uniformity.